scripts/perlang-install

```diff
@@ -100,14 +105,30 @@ main() {
         nightly)
             _toolchain_slug=latest
             ;;
+        release)
+            ;;
         *)
             err "Unsupported toolchain: $_toolchain"
             ;;
     esac
 
     info "downloading toolchain '$_toolchain' for ${_arch}"
 
-    local _url="${PERLANG_BUILDS_ROOT}/perlang-${_toolchain_slug}-${_arch}.tar.gz"
+    if [ "$_toolchain" = "release" ]; then
+        local _metadata_url="https://raw.githubusercontent.com/perlang-org/perlang/master/.metadata/latest-release.txt"
+
+        # mktemp is not POSIX, but exists on at least Linux and FreeBSD. (How about Git Bash on Windows?)
```

Comment:
Hmm, in fact not, since we only test installing the nightlies... :see_no_evil: https://github.com/perlang-org/perlang/runs/5349070448?check_suite_focus=true

We should expand the CI coverage here to install both the releases and the nightlies. Running CI for releases on each `master` commit feels a bit overkill though...

Reply:
Created an issue for this so that we can deal with this in the near future: https://github.com/perlang-org/perlang/issues/294.